velox/functions/prestosql/ArrayIntersectExcept.cpp

```diff
@@ -130,15 +143,9 @@ class ArrayIntersectExceptFunction : public exec::VectorFunction {
     auto baseLeftArray = decodedLeftArray->base()->as<ArrayVector>();
 
     // Decode and acquire array elements vector.
-    auto leftElementsVector = baseLeftArray->elements();
-    auto leftElementsRows = toElementRows(
-        leftElementsVector->size(),
-        rows,
-        baseLeftArray,
-        decodedLeftArray->indices());
-    exec::LocalDecodedVector leftElementsHolder(
-        context, *leftElementsVector, leftElementsRows);
-    auto decodedLeftElements = leftElementsHolder.get();
+    exec::LocalDecodedVector leftElementsDecoder(context);
```

Comment:
This looks like an unrelated refactoring. It would be nice to put it into a separate PR to allow the review to focus on material changes.

Reply:
@mbasmanova Can we retain this for now. Since I have already moved out common code and refactored other functions, there is too much refactoring to undo. Apologies, will make sure future PRs are concise.